thoth/package_extract/image.py

```diff
@@ -356,5 +384,6 @@ def run_analyzers(path: str, timeout: int = None) -> dict:
         "rpm-dependencies": _run_rpm_repoquery(path, timeout=timeout),
         "deb": deb_packages,
         "deb-dependencies": _run_apt_cache_show(path, deb_packages, timeout=timeout),
-        "python-files": _gather_python_file_digests(path)
+        "python-files": _gather_python_file_digests(path),
+        "operating-system": _gather_os_info(path),
```

Comment:
We expect this path to have the value ```"/etc/os-release"```, but isn't it the location to the temporary folder where image is extracted something like ```temporary_file/rootfs/```?

Reply:
We create a path relative to the extracted image path:
 
```python
    os_release_file = Path(path) / "/etc/os-release"
```

See the actual function implementation.

Reply:
Thanks @fridex for the explanation!